app/src/main/res/values-night/colors.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
```

Comment:
A couple issues to fix here (and in the light version too):

- The color lists are now a mix of exact color name (eg. `pink100`) and context (eg. `icon`), which is confusing. The exact denominations having different values between light/dark variants is especially odd as is completely defeats the point of using denominations in the first place. Now considering how the `DayNight` theme works, I understand the color names need to be shared between the light and dark files. As a result, I would prefer migrating everything to using contextual names like `background`, `icon`, `primary`, etc. It's ok if you need to introduce duplicate colors because of this.
- All colors should be picked from the [material design colors](https://www.materialui.co/colors). Since we are no longer going to use these denominations as the color names, I would at least like to have them as comments next to their usage. eg: `<color name="background">#212121</color> <!--Grey900-->`
- If you see opportunities to remove colors and use values already defined in the `DayNight` theme, that is also good!